Services/Component/classes/class.ilObjComponentSettingsGUI.php

```diff
@@ -517,17 +502,28 @@ function updatePlugin()
 		$pl = ilPlugin::getPluginObject($_GET["ctype"], $_GET["cname"],
 			$_GET["slot_id"], $_GET["pname"]);
 
-		$result = $pl->update();
-		
+		$this->update($pl);
+	}
+
+	/**
+	 * Execute update and ctrl reload
+	 *
+	 * @param ilPlugin 	$plugin
+	 *
+	 * @return null
```

Comment:
You know I love nitpicking, so go on =)

I just wanted to understand the use case. The thing with PHP 7.1 and PHPStorm is absolutely enough for me. We might want to add documentation guidelines (or even an automatic linter for these) and then should remember this again. Basically `void` seems to say "I really do not intend to return anything." while `null` says "I mean to return just one value, which is null.". If PHP 7.1 understands this difference we could as well introduce it.